socorro/cron/jobs/reprocessingjobs.py

```diff
@@ -30,10 +31,24 @@ class ReprocessingJobsApp(BaseCronApp):
     app_version = '0.1'
 
     def run(self, connection):
+        select_sql = """
+            SELECT crash_id FROM reprocessing_jobs LIMIT 10000
+        """
+        crash_ids = []
+        for crash_id, in execute_query_iter(connection, select_sql):
+            crash_ids.append(crash_id)
+
+        delete_sql = """
+            DELETE from reprocessing_jobs WHERE crash_id = %(crash_id)s
+        """
 
-        for crash_id, in execute_query_iter(connection, _reprocessing_sql):
+        for crash_id in crash_ids:
             self.queuing_connection_factory.save_raw_crash(
                 DotDict({'legacy_processing': 0}),
                 [],
                 crash_id
             )
+            execute_no_results(connection, delete_sql, {
+                'crash_id': crash_id
+            })
+            connection.commit()
```

Comment:
I think we can find a compromise. Can we try committing every 100 submitted crashes? This way, if the postgres connection fails, we let that bubble up, and next time crontabber re-runs this job we know that only at most 100 crashes will be re-reprocessed. 

Note: this is based on a gut feeling that committing 10,000 times to postgres is a bad idea, but maybe it isn't such a big deal. I don't know. :)